README.md

```diff
@@ -265,7 +253,7 @@ docker build -t trailofbits/polytracker-demo-mupdf -f examples/pdf/Dockerfile-mu
 ```
 
 `mutool_track` will be build in `/polytracker/the_klondike/mupdf/build/debug`. Running `mutool_track` will output
-`polytracker.db` which contains the information provided by the taint analysis. Its recommended to use this json with
+`polytracker.tdag` which contains the information provided by the taint analysis. Its recommended to use this json with
```

Comment:
it is not a json-file, that is some legacy wording

Reply:
Should I also remove the recommendation to use Polyfile, since it does not support `.tdag`?

Reply:
Yes, I think so. At least for now, until we've added support for tdag in polyfile.

Reply:
Fixed